src/common/config/defaultPreferences.js

```diff
@@ -9,7 +9,9 @@ const defaultPreferences = {
   trayIconTheme: 'light',
   minimizeToTray: false,
   notifications: {
-    flashWindow: 0
+    flashWindow: 0,
+    bounceIcon: 0,
```

Comment:
The default value should be `false` for boolean preferences.

Reply:
Ah indeed, I've changed it!